docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
     <<: *holdings-container-defaults
     restart: always
     volumes:
+      - .:/usr/src/app
+      - gem_cache:/gems
```

Comment:
Do we want to add gem caching here and now? 
Fine if so, just was not expecting to see it, wasn't mentioned in the PR desc.

Reply:
Gem caching is not new (it was being used in dev/test here), it was just missing in sidekiq_web, and sidekiq_web would not have worked for some time. I will revise the commit message to add a note about this.